generators/app/templates/src/app/screens/Login/layout.tsx

```diff
@@ -38,16 +45,24 @@ function Login({ onEmailChange, onPasswordChange, onLogin }: Props) {
           inputClassName={`m-bottom-2 full-width ${styles.input}`}
           placeholder={i18next.t('Login:passwordPlaceholder') as string}
           onChange={onPasswordChange}
+          disabled={loading}
         />
       </div>
       <div className={`column center ${styles.sectionContainer}`}>
-        <button type="submit" className={`full-width m-bottom-1 ${styles.button}`}>
+        <button disabled={loading} type="submit" className={`full-width m-bottom-1 ${styles.button}`}>
           {i18next.t('Login:enter')}
         </button>
+        <span
+          className={`row center middle full-width m-top-1 m-bottom-1
+          ${isError ? styles.errorMessage : styles.hidden}`}
```

Comment:
do we want to hide it from css or make it disapear from the dome? if css is the final decision make the class global.